frontend/src/app/modules/landing/landing.component.html

```diff
@@ -0,0 +1,28 @@
+<img class="logo" src="/assets/OpenSpeedMonitor_dark.svg" alt="OpenSpeedMonitor"/>
+
+<main *ngIf="hasData$ | async" [class.center]="showApplicationEmptyState$ | async">
```

Comment:
Do we need `hasData$`? `showApplicationEmptyState$` seems to be sufficient for me.

Reply:
`hasData$` assures the content isn't rendered at all until we know **if** and *which* applications we can list. Otherwise an empty card would be rendered that may "jumps" to the center after loading and no applications are available.